src/simulate/main.py

```diff
@@ -42,31 +46,35 @@ def main():
 
     files = os.listdir(json_files_dir)
     logging.info('list of json files in the directory %s', files)
-    for file in files:
-        path_to_file = json_files_dir + "/" + file
-        try:
-            caliper_event = open(path_to_file, 'rb')
-        except IOError as e:
-            logging.error('cannot read the file %s due to %s', path_to_file, e)
-            continue
-        with caliper_event:
-            event = caliper_event.read()
+    for i in range(run_count):
+        for file in files:
+            path_to_file = json_files_dir + "/" + file
             try:
-                jsonEvent = json.loads(event)
-            except JSONDecodeError as e:
-                logging.error('Failed to Deserialize the caliper event %s ', e)
+                caliper_event = open(path_to_file, 'rb')
+            except IOError as e:
+                logging.error('cannot read the file %s due to %s', path_to_file, e)
                 continue
-            # make needed changes to the json events
-            event_transformer = Transformer(jsonEvent, config_yml_obj)
-            json_event_transformed = event_transformer.event_transformer()
+            with caliper_event:
+                event = caliper_event.read()
+                try:
+                    jsonEvent = json.loads(event)
+                except JSONDecodeError as e:
+                    logging.error('Failed to Deserialize the caliper event %s', e)
+                    continue
+                # make needed changes to the json events
+                event_transformer = Transformer(jsonEvent, config_yml_obj)
+                json_event_transformed = event_transformer.event_transformer()
 
-            if json_event_transformed is None:
-                logging.error('Problem in transforming a event Json')
-                continue
-            # sending to endpoint
-            handler = HttpHandler(config_yml_obj)
-            handler.make_api_call(json_event_transformed)
-    logging.info('End Of App')
+                if json_event_transformed is None:
+                    logging.error('Problem in transforming a event Json')
+                    continue
+                # sending to endpoint
+                handler = HttpHandler(config_yml_obj)
+                handler.make_api_call(json_event_transformed)
+                sleep(random.uniform(0.5, 1))
+        logging.info('running count %s ', i + 1)
+
+    logging.info('End Of Script')
```

Comment:
Back to scripting again?  😉

Reply:
I think that got back on when resolving conflicts manually. Removed now

Reply:
Although I added a wink, my comment wasn't 100% humorous.

If problems you've corrected in earlier PRs of other branches reappear later, we need to be very concerned that you may be having trouble using git.  Logging a message about "script" vs. "app" is innocuous, but it makes me wonder what other changes have been reverted that we didn't notice?

Great care needs to be taken when making new issue branches that depend on changes from other issue branches.  I'm not sure whether your trouble comes from your making new branches from other branches.  Trouble could possibly come even if you usually make branches from master instead, but I think that would be safer.

I suspect the problem is that you're not pulling from GitHub to your branches often enough.